### ISSUE TITLE
README: "open your issue against"

### ISSUE THREAD

**Issue by Chealer** [eclipse-mylyn/.github#34]
[The README's _Reporting issues_ section](https://github.com/eclipse-mylyn#reporting-issues) ends with the following paragraph:

> If you face an issue and know which particular GitHub repository is most related, you can open your issue against that repository. If unsure, [you can open an issue here](https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues).

An issue can be _introduced_. _solved_ or _discovered_, but not "opened" (as opposed to a _ticket_).

This should read something like:
> If you face an issue and know which particular GitHub repository is most related, you can **report** your issue against that repository. If unsure, [you can file an issue **report** here](https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues).

<details><summary>By the way</summary>
<p>

1. The list may be misleading, as there is also a repository for the website.
2. The section should be retitled "Tracking issues"/"Issue tracking", or just "Issues" (since hopefully the issue will already be reported).
3. The "Issues" link(s) from [the _Jump in_ section](https://github.com/eclipse-mylyn/.github/blob/main/profile/README.md#jump-in) should be moved to this section.
</p>
</details>

**Comment by wimjongman** [comment]
Thanks for pointing this out. I guess that when we say _issue_ we mean _issue report_. One could argue that the term _issue_ has become equivalent to the term _ticket_, which we used to use.  Strictly speaking, you can also not _open_ a _ticket_.  

Please go ahead and change the content so that it is better.

**Comment by Chealer** [comment]
Hi @wimjongman 

> Thanks for pointing this out.

An honour

> I guess that when we say _issue_ we mean _issue report_.

In some instances, clearly.

> One could argue that the term _issue_ has become equivalent to the term _ticket_, which we used to use. Strictly speaking, you can also not _open_ a _ticket_.

Hum. Do you agree that a ticket can be closed?
 
> Please go ahead and change the content so that it is better.

I think the formulation I suggested (without the emphasis) would be fine.

**Comment by wimjongman** [comment]
> Hum. Do you agree that a ticket can be closed?

Yes, if you say that a Ticket now means a bug report. A ticket used to be something that you bought to get on a train or something ;). Semantics... 🗡️

**Comment by Chealer** [comment]
### "Ticket"

> > Hum. Do you agree that a ticket can be closed?
> 
> Yes, if you say that a Ticket now means a bug report.

I wouldn't go that far as "ticket" didn't lose its original sense. But "ticket" *can* mean "issue report", or something very close to that (which I would basically describe as a recorded change request).

> A ticket used to be something that you bought to get on a train or something ;). Semantics... 🗡️

Indeed, but the word complexified a long time ago. You will find plenty usages of "(IT/support/helpdesk) ticketing system" online even from their creators, and out of 3 dictionaries I consulted:
* Wiktionary acknowledges "A service request, used to track complaints or requests that an issue be handled. (Generally technical support related)."
* Cambridge dictionary acknowledges "a request sent online or by phone for help with a problem with a website, computer system, etc.: "
* The only one which doesn't list that sense does list at least 1 non-physical sense.

And none of these indicate the sense as discouraged.

I am convinced that "ticket" can be used as a quasi-synonymous for "issue report", but I do not recommend it over "issue report".

**Comment by wimjongman** [comment]
Yes, I agree.
